wgpu-core/src/device/mod.rs

```diff
@@ -70,6 +70,12 @@ impl<T> AttachmentData<T> {
     }
 }
 
+#[derive(Debug, Copy, Clone)]
+pub enum RenderPassCompatibilityCheckType {
+    RenderPipeline,
+    RenderBundle,
+}
```

Comment:
Won't the debug impl print the name of the enum as well? Might be too verbose for error messages.

Reply:
I was worried it would to, but it just prints the enum variant name